app/keyboard_monitor.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Google Inc.
```

Comment:
Let's land this PR with out app/controller.py changes or app/keyboard_monitor.py. The pynput library looks like it's GPL, that can be problematic in some uses. If we do two PRs we can land the other changes sooner.

Reply:
Hmm, yeah I see the conflicts with the licenses. I'll have to see if it's possible to do the detection myself or find some alternative library..

Reply:
I'm happy to land the rest of this CL if you're up for clipping the app/controller.py and app/keyboard_monitor.py changes (the other changes have no license issues).